package.json

```diff
@@ -1,24 +1,91 @@
 {
-  "name": "apply-loader",
-  "version": "1.0.0",
-  "description": "Apply loader for webpack",
-  "main": "index.js",
+  "_args": [
+    [
+      {
+        "raw": "apply-loader",
+        "scope": null,
+        "escapedName": "apply-loader",
+        "name": "apply-loader",
+        "rawSpec": "",
+        "spec": "latest",
+        "type": "tag"
+      },
+      "/Users/ish-/Documents/DEV/mm-web"
+    ]
+  ],
+  "_from": "apply-loader@latest",
+  "_id": "apply-loader@1.0.0",
+  "_inCache": true,
+  "_location": "/apply-loader",
+  "_nodeVersion": "7.2.1",
+  "_npmOperationalInternal": {
+    "host": "packages-18-east.internal.npmjs.com",
+    "tmp": "tmp/apply-loader-1.0.0.tgz_1485360869540_0.09120802441611886"
+  },
+  "_npmUser": {
+    "name": "mogelbrod",
+    "email": "victor@hallberg.cc"
+  },
+  "_npmVersion": "3.10.9",
+  "_phantomChildren": {},
+  "_requested": {
+    "raw": "apply-loader",
+    "scope": null,
+    "escapedName": "apply-loader",
+    "name": "apply-loader",
+    "rawSpec": "",
+    "spec": "latest",
+    "type": "tag"
+  },
+  "_requiredBy": [
+    "#DEV:/",
+    "#USER"
+  ],
+  "_resolved": "https://registry.npmjs.org/apply-loader/-/apply-loader-1.0.0.tgz",
+  "_shasum": "f14e9f0d4b031ddf27b8a017083559ff1db5ad3e",
+  "_shrinkwrap": null,
+  "_spec": "apply-loader",
+  "_where": "/Users/ish-/Documents/DEV/mm-web",
```

Comment:
Could you remove all the underscore prefixed props? They don't look very official :)